.github/workflows/default_workflow.yml

```diff
@@ -0,0 +1,52 @@
+name: Default Workflow
+
+concurrency:
+  group: wf-${{github.event.pull_request.number || github.sha}}-${{github.workflow}}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "[0-9][0-9]_[0-9]"
+
+env:
+  NX_SKIP_NX_CACHE: ${{ (github.event_name != 'pull_request' || contains( github.event.pull_request.labels.*.name, 'skip-cache')) && 'true' || 'false' }}
+
+jobs:
+  main:
+    name: main
+    runs-on: devextreme-shr2
+    timeout-minutes: 15
+    
+    steps:
+    - name: Get sources
+      uses: actions/checkout@v4
+    
+    - name: Restore npm cache
+      uses: actions/cache@v4
+      with:
+        path: '**/node_modules'
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: ${{ runner.os }}-node-modules
+
+    - name: Run npm install
+      run: npm ci --no-audit --no-fund
+    
+    - name: Run targets
```

Comment:
**Minor**
```suggestion
    - name: Test
```

Reply:
In future there will be different target, such as lint, compile-ts and so on

Reply:
Why not extracting each target's `run-many` as a workflow step? Looks much more convenient

Reply:
If we specify them in one `nx` command, they will run in parallel
If in separate step, each target will wait for previous to finish (because next step of workflow doesn't start until previous is finished)

Reply:
It's nx's guideline to specify all commands in one shell command to reduce unnecessary waits (no need to wait for build to lint; no need to build qunit to run testcafe etc)

Reply:
Got it. Thank you for the explantion!